### DISCUSSION TITLE
Form Field Tooltips (with icon)

### DISCUSSION THREAD

**Discussion by diogenesbrussels** [Budibase/budibase#7852] [Ideas]
Hi everyone,

I regularly find the need to add clarifications or a little help regarding the fields in the forms.
I think it would be very useful to add a tooltip helper property to the form fields (for example like here: https://uxmovement.com/mobile/how-to-display-tooltips-on-mobile-forms/)

At the moment the only solution is long and impractical: Add a container, place the field and an icon element in it and then add an onclick event, etc. which does not give a satisfactory result.

I thought of two solutions to add this:
- Add a Tooltip helper option in the field properties that if completed adds a question mark icon to the field (clickable or hoverable depending on the device type)
- Add a 'on hover'/tooltip property to icon elements and allow this element to be placed inside field elements.

What do you think of this idea?

Thanks for your help/feedback!

**Comment by melohagan** [comment]
Love this idea! There's an open feature request here: https://github.com/Budibase/budibase/issues/5743
